vip-integrations.php

```diff
@@ -14,16 +14,22 @@
 
 defined( 'ABSPATH' ) || die();
 
+// @codeCoverageIgnoreStart - the actual code here is tested individually in the unit tests.
+
 require_once __DIR__ . '/integrations/integration.php';
 require_once __DIR__ . '/integrations/integrations.php';
 require_once __DIR__ . '/integrations/enums.php';
 require_once __DIR__ . '/integrations/integration-vip-config.php';
 require_once __DIR__ . '/integrations/block-data-api.php';
 require_once __DIR__ . '/integrations/parsely.php';
+require_once __DIR__ . '/integrations/vip-governance.php';
 
 // Register VIP integrations here.
 IntegrationsSingleton::instance()->register( new BlockDataApiIntegration( 'block-data-api' ) );
 IntegrationsSingleton::instance()->register( new ParselyIntegration( 'parsely' ) );
+IntegrationsSingleton::instance()->register( new VipGovernanceIntegration( 'vip-governance' ) );
```

Comment:
Due to deployed not being atomic, we need to safeguard this just for the initial deploy. Something like this:

```
if ( file_exists( __DIR__ . '/integrations/vip-governance.php' ) ) {
	require_once __DIR__ . '/integrations/vip-governance.php';
	IntegrationsSingleton::instance()->register( new VipGovernanceIntegration( 'vip-governance' ) );
}
```

Reply:
I moved that line, and the initial require under the if check. Hopefully that should safeguard the code for the initial deploy. Lemme know if that doesn't look right.

Reply:
Looks good now!